.github/workflows/deploy-js-kit-client.yml

```
@@ -31,37 +36,74 @@ env:
-    name: JS client
-    needs: build_programs
-    uses: ./.github/workflows/test-js-client.yml
+    needs: check_tag
+    if: needs.check_tag.outputs.type == 'ref'
+    with:
+      git_ref: ${{ inputs.git_ref }}
+
+  test_js_kit:
+    name: JS Kit client
+    needs: [build_programs, check_tag]
+    uses: ./.github/workflows/test-js-kit-client.yml
```

actionlint (1.7.11)

[error] could not read reusable workflow file for "./.github/workflows/test-js-kit-client.yml": open /home/jailuser/git/.github/workflows/test-js-kit-client.yml: no such file or directory

(workflow-call)